content/post/Matter_future.md

```diff
@@ -0,0 +1,127 @@
++++
+title = 'Matter and the future'
+slug = 'Matter_future'
+image = '/static/images/Cover.jpeg'
+date = "2023-03-23T10:00:00"
+author = 'Tushar Bhatia'
+tags=['IoT', 'Networking', 'Smart Home', 'Smart Devices', 'Google-projects']
+categories =['IoT', 'Smart-Devices', 'Smart Homes', 'Networks']
+description = 'This blog explores Matter a new communication protocol made by CSA'
+disableComments = false
++++
+
+Hi everyone, This blog explores the new communication protocol, known as Matter which is being developed by <strong>Connectivity Standards Alliance (CSA)</strong>
+<br>
+To know more, Let's go on a journey and explore the new generation of communication protocols.
+<br>
+
+  ## What's Matter?
+  As the IoT market boomed. The popularity of smart home devices shot up. This resulted in a new market for Smart Home. Many companies joined hands and contributed to this boom by making Smart Devices.
+  <br>
```

Comment:
You don't need `<br>` or `<p>` with markdown, just add 2 newlines wherever you want to start a new paragraph.